resources/references/core-reference/administration-reference/utils.md

```diff
@@ -0,0 +1,109 @@
+# Utils reference
+
+This is an overview of all the utility functions bound to the shopware global object.
+Utility functions provide many useful shortcuts for common tasks, see how to use them in your plugin [here](./../../../../guides/plugins/plugins/administration/using-utils.md).
+
+## General functions
+
+| Function | Description | Link
+| :--- | :--- | :---- |
+| createId | Returns a uuid string in hex format. Generated with [uuid](https://www.npmjs.com/package/uuid)
+| throttle | Creates a `throttled` function that only invokes `func` at most once per every `wait` milliseconds. | [link](https://lodash.com/docs/4.17.15#throttle)
+| debounce | Creates a `debounced` function that delays invoking `func` until after `wait` milliseconds have elapsed since the last time the `debounced` function was invoked. | [link](https://lodash.com/docs/4.17.15#debounce)
+| flow     | Creates a function that returns the result of invoking the given functions with the `this` binding of the created function, where each successive invocation is supplied the return value of the previous. | [link](https://lodash.com/docs/4.17.15#flow)
+| get      | Gets the value at `path` of `object` | [link](https://lodash.com/docs/4.17.15#get) |
+
+## Object
+
+| Function | Description | Link
+| :--- | :--- | :---- |
+| deepCopyObject  | Deep copy an object |
+| hasOwnProperty  | Shorthand method for `Object.prototype.hasOwnProperty` |
+| getObjectDiff   | Gets a simple recursive diff of two objects. Does not consider an entity schema or entity related logic. |
+| getArrayChanges | Check if the compared array has changes. |
+| cloneDeep       | Creates recursivly a clone of value. | [link](https://lodash.com/docs/4.17.15#cloneDeep)
+| merge           | This method is like _.assign except that it recursively merges own and inherited enumerable string keyed properties of source objects into the destination object. | [link](https://lodash.com/docs/4.17.15#merge)
+| mergeWith       | This method is like _.merge except that it accepts customizer which is invoked to produce the merged values of the destination and source properties. | [link](https://lodash.com/docs/4.17.15#mergeWith) 
+| deepMergeObject | Deep merge two objects |
+| get             | Gets the value at `path` of `object` | [link](https://lodash.com/docs/4.17.15#get) |
+| set             | Sets the value at `path` of `object` | [link](https://lodash.com/docs/4.17.15#set) |
+| pick            | Creates an object composed of the picked `object` properties. | [link](https://lodash.com/docs/4.17.15#pick) |
+
+## Debug
+
+| Function | Description |
+| :--- | :--- |
+| warn | General logging function which provides a unified style of log messages for developers. Please keep in mind the log. Messages will be displayed in the developer console when they're running the application in development mode. |
```

Comment:
Please keep in mind the log ? Is this a finished sentence ?

Reply:
Maybe he meant this one:

```suggestion
| warn | General logging function which provides a unified style of log messages for developers. Please keep the log in mind. Messages will be displayed in the developer console when they're running the application in development mode. |
```